The actor creation benchmark and Subdirectories

Wonseo-C opened the pull request:
* https://github.com/lf-lang/benchmarks-lingua-franca/issues/23 's first task (organizing subdirectories)
* I Implemented <code>ActorCreation</code> by using Mutation (https://github.com/lf-lang/lingua-franca/pull/1252)

hokeun commented:
Looks like the ts-benchmark-tests are failing:

Here is the raw error message:

```
Error executing job with overrides: ['test_mode=True', 'iterations=1', 'benchmark=savina_concurrency_philosopher', 'target=lf-ts']
Traceback (most recent call last):
  File "/home/runner/work/benchmarks-lingua-franca/benchmarks-lingua-franca/runner/run_benchmark.py", line 75, in main
    check_return_code(code, continue_on_error)
  File "/home/runner/work/benchmarks-lingua-franca/benchmarks-lingua-franca/runner/run_benchmark.py", line [107](https://github.com/lf-lang/benchmarks-lingua-franca/runs/7400539022?check_suite_focus=true#step:17:108), in check_return_code
    raise RuntimeError(
RuntimeError: Command returned with non-zero exit code (1)
Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
Error: Process completed with exit code 1.
```

I think there is an issue with running the `savina_concurrency_philosopher` benchmark.

Here is a screenshot just for your information.
![Screen Shot 2022-07-18 at 5 43 53 PM](https://user-images.githubusercontent.com/2585943/179640255-d1789448-b768-4e01-8658-efe9b48852ac.png)

Wonseo-C commented:
@hokeun I think the test was passed (by modifying <code>savina_concurrency_philosopher.yaml</code>)!

cmnrd reviewed (commented):
Great! The only comment I have: `NodeHTTPServer.js` is a custom benchmark and not part of the Savina suite. I think we can just leave it where it was.

Wonseo-C commented:
I changed the PingPong benchmark by resetting the message counter. (commit -> https://github.com/lf-lang/benchmarks-lingua-franca/pull/28/commits/51b19c55fa2c65d53cd26c959557e1bc037ca314)
And also, I added the lf-ts for the PingPong benchmark. (commit -> https://github.com/lf-lang/benchmarks-lingua-franca/pull/28/commits/4d47ff3965223e6ff1ceb2748cba340de4b40955)
@cmnrd (https://github.com/lf-lang/benchmarks-lingua-franca/pull/21; I resolved this issue, so I closed the PR)

cmnrd reviewed (approved):
Looks like good progress!
